mealpy.py

```diff
@@ -110,7 +116,8 @@ def reserve_meal(
             'source': 'Web',
         }
 
-        request = requests.post(RESERVATION_URL, data=json.dumps(reserve_data), headers=HEADERS, cookies=self.cookies)
+        request = requests.post(RESERVATION_URL, data=json.dumps(reserve_data),
+                                headers=HEADERS, cookies=self.cookies)
```

Comment:
What's the stylistic choice here? The line-length wasn't exceeded.

Reply:
Reverting, my IDE line guide was set at 79 instead of 120.

Reply:
FYI, I set up pre-commit hooks that you can install with `make venv`. It runs `flake8` which will lint for line-length from `setup.cfg`.

Reply:
Just did it, works good now!